src/IssueViz/Editor/LineHashCalculator.cs

```diff
@@ -32,16 +31,13 @@ public interface ILineHashCalculator
         /// <param name="documentText">The text from which to extract the line</param>
         /// <param name="oneBasedLineNumber">1-based line to hash</param>
         /// <returns>hash of line</returns>
-        string Calculate(string documentText, int oneBasedLineNumber);
+        string Calculate(ITextDocument documentText, int oneBasedLineNumber);
```

Comment:
The line hash calculator shouldn't be responsible for choosing which snapshot to use. It would be better if the snapshot was supplied by the caller.